src/components/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,187 @@
+@import '../../styles/colors';
+
+.Accordion {
+  --zui-accordion-margin-top: 0.25rem;
+  --zui-accordion-group-border-radius: 0.5rem;
+
+  .AccordionItem:first-child {
+    --zui-accordion-margin-top: 0;
+  }
+
+  .AccordionHeader {
+    --zui-accordion-border-bottom: 0.063rem solid #{$purple-50};
+
+    &:hover {
+      --zui-accordion-border-bottom: 0.063rem solid #{$purple-8};
+    }
+
+    &:active {
+      --zui-accordion-border-bottom: 0.063rem solid #{$purple-50};
+    }
+  }
+
+  .AccordionTrigger {
+    --zui-accordion-item-padding: 0.5rem 0 0.438rem;
+    --zui-accordion-header-background-color: transparent;
+
+    &:hover {
+      --zui-accordion-header-background-color: #{$purple-2};
+      --zui-accordion-item-padding: 0.5rem 0px 0.438rem;
+    }
+
+    &[data-state='open'] {
+      --zui-accordion-header-background-color: #{$purple-2};
+      --zui-accordion-item-padding: 0.5rem 1rem 0.438rem;
+
+      &:hover {
+        --zui-accordion-header-background-color: #{$purple-2};
+        --zui-accordion-item-padding: 0.5rem 1rem 0.438rem;
+      }
+    }
+
+    &:active {
+      --zui-accordion-header-background-color: #{$purple-2};
+    }
+  }
+
+  &[data-contrast='high'] {
+    .AccordionHeader {
+      --zui-accordion-border-bottom: 0.063rem solid #{$purple-6};
+
+      &[data-state='open'] {
+        --zui-accordion-border-bottom: none;
+      }
+
+      &:hover {
+        --zui-accordion-border-bottom: 0.063rem solid #{$purple-8};
+      }
+
+      &:active {
+        --zui-accordion-border-bottom: 0.063rem solid #{$purple-6};
+      }
+    }
+
+    .AccordionTrigger {
+      --zui-accordion-header-background-color: #{$purple-4};
+      --zui-accordion-item-padding: 0.5rem 1rem 0.438rem;
+
+      &[data-state='open'] {
+        --zui-accordion-header-background-color: #{$purple-2};
+      }
+
+      &:hover {
+        --zui-accordion-header-background-color: #{$purple-2};
+      }
+    }
+  }
+}
+
+.Accordion {
```

Comment:
I think we can simplify this file. Could we group styles together to improve readability. We should be able to declare classNames once and nest the styles

Reply:
As per our conversation, we keeping the variables.